src/openrct2/ride/Track.cpp

```diff
@@ -197,7 +197,7 @@ ResultWithMessage TrackAddStationElement(CoordsXYZD loc, RideId rideIndex, int32
         return { false, STR_NO_MORE_STATIONS_ALLOWED_ON_THIS_RIDE };
     }
 
-    if (stationLength > MAX_STATION_PLATFORM_LENGTH)
+    if (stationLength > kMaxStationPlatformLength)
```

Comment:
`stationLength` is signed, so we should use `int8_t`

Reply:
Should we just make it `int32_t` exactly as `stationLength`?

Reply:
I think there is no need to make it `int32_t`, since it's smaller, but signedness should agree.

Reply:
Committed changes, now it is a signed `int8_t`.